scripts/toolchains/uwp.cmake

```
@@ -0,0 +1,61 @@
+if(NOT _VCPKG_WINDOWS_TOOLCHAIN)
+set(_VCPKG_WINDOWS_TOOLCHAIN 1)
+set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>$<$<STREQUAL:${VCPKG_CRT_LINKAGE},dynamic>:DLL>" CACHE STRING "")
+
+get_property( _CMAKE_IN_TRY_COMPILE GLOBAL PROPERTY IN_TRY_COMPILE )
+if(NOT _CMAKE_IN_TRY_COMPILE)
+
+    if(VCPKG_CRT_LINKAGE STREQUAL "dynamic")
+        set(VCPKG_CRT_LINK_FLAG_PREFIX "/MD")
+    elseif(VCPKG_CRT_LINKAGE STREQUAL "static")
+        set(VCPKG_CRT_LINK_FLAG_PREFIX "/MT")
+    else()
+        message(FATAL_ERROR "Invalid setting for VCPKG_CRT_LINKAGE: \"${VCPKG_CRT_LINKAGE}\". It must be \"static\" or \"dynamic\"")
+    endif()
+
+    set(_vcpkg_charset "/utf-8")
+    if (NOT VCPKG_SET_CHARSET_FLAG OR VCPKG_PLATFORM_TOOLSET MATCHES "v120")
+        # VS 2013 does not support /utf-8
+        set(_vcpkg_charset "")
+    endif()
+
+    set(_vcpkg_cpp_flags "/DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__" ) # VS adds /D "_WINDLL" for DLLs;
+    set(_vcpkg_common_flags "/nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc") 

```

```suggestion
    set(_vcpkg_common_flags "/nologo /Z7 /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc") 
```

`/MP` is a workaround for an MSBuildism that doesn't apply here.

---

(Don't worry about applying this yet; I'll just apply it before merging. Just avoiding building the world again until I'm ready to push the merge button)